This test is failing because it assumes to be run in it's own repository.

See https://github.com/jeffkaufman/icdiff/issues/166
diff --git a/test.sh b/test.sh
index 99729d7234..bad21cd213 100755
--- a/test.sh
+++ b/test.sh
@@ -144,7 +144,6 @@ check_gold gold-subcolors-bad-color tests/input-{1,2}.txt --cols=80 --color-map=
 check_gold gold-subcolors-bad-cat tests/input-{1,2}.txt --cols=80 --color-map='chnge:magenta,description:cyan_bold'
 check_gold gold-subcolors-bad-fmt tests/input-{1,2}.txt --cols=80 --color-map='change:magenta:gold,description:cyan_bold'
 check_gold gold-bad-encoding.txt tests/input-{1,2}.txt --encoding=nonexistend_encoding
-check_git_diff gitdiff-only-newlines.txt 4e86205629~1 4e86205629
 
 # Testing pipe behavior doesn't fit well with the check_gold system
 $INVOCATION tests/input-{4,5}.txt 2>/tmp/icdiff-pipe-error-output | head -n 1
